src/GridFS/Bucket.php

```diff
@@ -551,6 +551,7 @@ private function getRawFileDocumentForStream($stream)
         if ( ! isset ($metadata['wrapper_data']) || ! $metadata['wrapper_data'] instanceof StreamWrapper) {
             throw InvalidArgumentException::invalidType('$stream wrapper data', isset($metadata['wrapper_data']) ? $metadata['wrapper_data'] : null, 'MongoDB\Driver\GridFS\StreamWrapper');
         }
+        $metadata['wrapper_data']->stream_close();
```

Comment:
This appears to be a breaking change as it subtly closes the stream when [`Bucket::getFileDocumentForStream()`](https://github.com/mongodb/mongo-php-library/blob/1.2.0-alpha1/src/GridFS/Bucket.php#L299) is called where currently the stream is left in its current state.